0_fund_processing.R

```diff
@@ -0,0 +1,80 @@
+library(tidyverse)
+library(xml2)
+
```

Comment:
I don't really understand what this script is doing in this repo tbh.. not necessarily relevant for this PR, but just saying @Clare2D